server/chat-plugins/mafia.ts

```diff
@@ -3463,12 +3473,14 @@ export const commands: ChatCommands = {
 			}
 			if (!gavePoints) return this.parse('/help mafia win');
 			writeFile(LOGS_FILE, logs);
-			this.modlog(`MAFIAPOINTS`, null, `${points} points were awarded to ${Chat.toListString(toGiveTo)}`);
+			this.modlog(`MAFIAPOINTS`, null, `${points < 0 ? points * -1 : points} points were ${points < 0 ? 'taken from' : 'awarded to'} ${Chat.toListString(toGiveTo)}`);
 			room.add(buf).update();
 		},
 		winhelp: [
 			`/mafia win (points), [user1], [user2], [user3], ... - Award the specified users points to the mafia leaderboard for this month. The amount of points can be negative to take points. Defaults to 10 points.`,
+			'/mafia unwin (points), [user1], [user2], [user3], ... - Take away the specified users points to the mafia leaderboard for this month. Defaults to -10 points.',
 			`/mafia winfaction (points), [faction] - Award the specified points to all the players in the given faction.`,
+			`/mafia unwinfaction (points), [faction] - Take away the specified points to all the players in the given faction`,
```

Comment:
you can probably just do `/mafia (un)win` and `/mafia (un)winfaction` instead of adding two separate lines